deploy/clowdapp.yaml

```diff
@@ -0,0 +1,85 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: notifications
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: notifications
+  spec:
+    envName: ${ENV_NAME}
+    deployments:
+    - name: notifications-backend
+      minReplicas: ${{MIN_REPLICAS}}
+      web: true
+      podSpec:
+        image: ${IMAGE}:${IMAGE_TAG}
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /
+            port: 8000
+            scheme: HTTP
+          initialDelaySeconds: 35
+          periodSeconds: 5
+          successThreshold: 1
+          timeoutSeconds: 120
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /
+            port: 8000
+            scheme: HTTP
+          initialDelaySeconds: 35
+          periodSeconds: 5
+          successThreshold: 1
+          timeoutSeconds: 120
+        env:
+        - name: CLOWDER_ENABLED
+          value: ${CLOWDER_ENABLED}
+        resources:
+          limits:
+            cpu: ${CPU_LIMIT}
+            memory: ${MEMORY_LIMIT}
+          requests:
```

Comment:
Can we also use parameters like it exists today:
https://github.com/RedHatInsights/e2e-deploy/blob/56d3c6990c7ba1250c536ed691cea45b140f65e3/templates/notifications/notifications-backend.yml#L135-L137

Reply:
Fixed

Reply:
I really meant to use parameters instead of hard coded 500m and 500Mi values

Reply:
Fixed